### PR TITLE
provider/aws: Exposed aws_iam_role create_date attribute

### LINKED ISSUE #10079: Documentation Update: missing outputs of aws_iam_role resource
Hi team TF,

I was able to get the following additional outputs from the _aws_iam_role_ resource (not shown in [docs](https://www.terraform.io/docs/providers/aws/r/iam_role.html)):

- path
- name

e.g. here's what `terraform show` gives me on a test _aws_iam_role_ resource:
```
aws_iam_role.test_role.role:
  id = kjh_test_app_name_kjh_test_app_id
  arn = arn:aws:iam::123456789123:role/kjh_test_app_name_kjh_test_app_id
  assume_role_policy = 
     {json policy}
  name = kjh_test_app_name_kjh_test_app_id
  path = /
  unique_id = AROAJUSUI3V5UPBW6D57G
```

And if I include some outputs for `aws_iam_role.test_role.name` and `aws_iam_role.test_role.path`, I get valid outputs.

Running some tests, it doesn't look like _create_date_ is also outputted. 

Unfortunately, Golang looks like Greek to me, and I didn't have the cycles to figure out what the code actually spits back.

Please close/delete if I have no idea what I'm talking about! Thanks!

-- Khalid

### PR DESCRIPTION
**Pull request by Ninir**
Fixed #10079

### CODE REVIEW

File: builtin/providers/aws/resource_aws_iam_role.go [hunk 1]
```diff
@@ -71,14 +71,19 @@ func resourceAwsIamRole() *schema.Resource {
...
 			},
 
-			"assume_role_policy": &schema.Schema{
+			"assume_role_policy": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"create_date": {
 				Type:     schema.TypeString,
```
Comment (stack72) [inline builtin/providers/aws/resource_aws_iam_role.go]: This should be `computed: true`, not `required: true`

**Comment by Ninir** [comment]
Hey @stack72 

Oops, failed the c/c... fixed. Sorry for the extra work.

**Comment by Ninir** [comment]
Tests passing.

**Comment by stack72** [comment]
This looks much better now :) Thanks for the work here!

```
% make testacc TEST=./builtin/providers/aws TESTARGS='-run=TestAccAWSRole_'
==> Checking that code complies with gofmt requirements...
go generate $(go list ./... | grep -v /terraform/vendor/)
2016/11/14 09:32:02 Generated command/internal_plugin_list.go
TF_ACC=1 go test ./builtin/providers/aws -v -run=TestAccAWSRole_ -timeout 120m
=== RUN   TestAccAWSRole_basic
--- PASS: TestAccAWSRole_basic (16.52s)
=== RUN   TestAccAWSRole_namePrefix
--- PASS: TestAccAWSRole_namePrefix (15.69s)
=== RUN   TestAccAWSRole_testNameChange
--- PASS: TestAccAWSRole_testNameChange (35.73s)
PASS
ok  	github.com/hashicorp/terraform/builtin/providers/aws	67.955s
```

**Pull request merged**